src/main/java/module-info.java

```diff
@@ -0,0 +1,6 @@
+module com.github.kmehrunes.javalinjwt {
+    exports javalinjwt;
+    requires io.javalin;
+    requires com.auth0.jwt;
+    requires org.jetbrains.annotations;
```

Comment:
Same comment as the dependency

Reply:
I am not sure what kind of comments do you expect but I added some that I thought would be help full.

Reply:
There's a misunderstanding, I meant I had the same comment about the annotations dependency but didn't want to repeat it. That's all